src/main/java/it/gov/pagopa/pu/workflow/wf/pagopa/paymentsreporting/PaymentsReportingPagoPaScheduler.java

```diff
@@ -0,0 +1,49 @@
+package it.gov.pagopa.pu.workflow.wf.pagopa.paymentsreporting;
+
+import io.temporal.client.schedules.*;
+import it.gov.pagopa.pu.workflow.service.WorkflowScheduleService;
+import it.gov.pagopa.pu.workflow.wf.pagopa.paymentsreporting.wfbrokersfetch.OrganizationsBrokeredRetrieveWF;
+import it.gov.pagopa.pu.workflow.wf.pagopa.paymentsreporting.wfbrokersfetch.OrganizationsBrokeredRetrieveWFImpl;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Value;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.time.Duration;
+
+@Slf4j
+@Component
+public class PaymentsReportingPagoPaScheduler {
+  private final WorkflowScheduleService workflowScheduleService;
+  private final Long scheduleDuration;
+  private final String scheduleId;
+
+  public PaymentsReportingPagoPaScheduler(WorkflowScheduleService workflowScheduleService,
+                                          @Value("${schedule.payments-reporting-pagopa.name}") String scheduleId,
+                                          @Value("${schedule.payments-reporting-pagopa.frequency-hour}") Long frequencyHour) {
+    this.workflowScheduleService = workflowScheduleService;
+    this.scheduleId = scheduleId;
+    this.scheduleDuration = frequencyHour;
```

Comment:
let's calculate here the Duration instead of perform this operation at each invocation
```suggestion
    this.scheduleDuration = Duration.ofHours(scheduleDuration);
```